arm-compute/2017-03-30/swagger/runCommands.json

```diff
@@ -198,6 +198,13 @@
           "type": "string",
           "description": "The run command id."
         },
+        "script": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The script to be executed."
```

Comment:
Here is related PR that includes the test of the feature.

https://github.com/Azure/azure-sdk-for-net/pull/3461